xtuner/engine/hooks/evaluate_chat_hook.py

```diff
@@ -160,9 +174,13 @@ def _generate_samples(self, runner, max_new_tokens=None):
                     max_new_tokens=max_new_tokens,
                     generation_config=self.gen_config,
                     stopping_criteria=self.stop_criteria)
-                runner.logger.info(
-                    f'Sample output:\n'
-                    f'{self.tokenizer.decode(generation_output[0])}\n')
+                generation_output = self.tokenizer.decode(generation_output[0])
+                runner.logger.info(f'Sample output:\n{generation_output}\n')
+                if save_eval_output:
+                    eval_outputs.append(f'{inputs + generation_output}\n')
```

Comment:
Do we need save `inputs` here？

Reply:
Save the evaluation results for gui display